lib/plugins/aws/package/compile/events/schedule.js

```diff
@@ -92,6 +96,7 @@ class AwsCompileScheduledEvents {
             let InputTransformer;
             let Name;
             let Description;
+            let Method;
```

Comment:
Let's use lower case.

I know it'll be inconsistent with other var names, but it's other var names that should be renamed (in JS var starting with capital, suggests function constructor, so it's highly confusing).

I'll be happy to take in other PR a refactor that renames all those vars to lowercase